dipy/denoise/tests/test_noise_estimate.py

```diff
@@ -110,6 +125,10 @@ def test_estimate_sigma():
 
     arr[0, 0, 0] = 1
     sigma = estimate_sigma(arr, disable_background_masking=True, N=12)
-    assert_array_almost_equal(sigma, np.array([0.46291005 / np.sqrt(0.4946862482541263),
-                                               0.46291005 / np.sqrt(0.4946862482541263),
-                                               0.46291005 / np.sqrt(0.4946862482541263)]))
+    assert_array_almost_equal(sigma,
```

Comment:
Should I change it to 
`assert_array_almost_equal(sigma, np.array([0.46291005 / np.sqrt(0.4946862482541263),
                                                                         0.46291005 / np.sqrt(0.4946862482541263),
                                                                         0.46291005 / np.sqrt(0.4946862482541263)]))`